internal/testbed/integration/zipkin/e2e_test.go

```diff
@@ -0,0 +1,226 @@
+//go:build e2e
+
+package zipkin
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/receiver/otlpreceiver"
+	"go.opentelemetry.io/collector/receiver/receivertest"
+	"go.uber.org/multierr"
```

Comment:
Nit: we can use `errors.Join` instead of `multierr` now.

Reply:
good suggestion, thanks - Since I have also a [refactoring PR](https://github.com/Dynatrace/dynatrace-otel-collector/pull/233) for the integration tests open right now, I will incorporate that in the refactoring PR, as this is one of the functions that have been changed/moved there